tensorboard/plugins/mesh/mesh_demo.py

```diff
@@ -0,0 +1,97 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Simple demo which displays constant 3D mesh."""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+
+from absl import app
+from absl import flags
+import numpy as np
+import tensorflow as tf
+
+
+from tensorboard.plugins.mesh import summary as mesh_summary
+from tensorboard.plugins.mesh import demo_utils
+
+flags.DEFINE_string('logdir', '/tmp/mesh_demo',
+                    'Directory to write event logs to.')
+flags.DEFINE_string('mesh_path', None, 'Path to PLY file to visualize.')
+
+FLAGS = flags.FLAGS
+
+tf.compat.v1.disable_v2_behavior()
```

Comment:
This is not expected to work at all in TF 2.0, right? Can we explicitly fail in that case, rather than leaving the user to deduce why `tf.placeholder` does not exist?

```python
if int(tf.__version__.split(".")[0]) > 1:
  # Mesh summaries only work on TensorFlow 1.x.
  raise ImportError("TensorFlow 1.x is required to run this demo.")
```